packages/bpk-docs/src/pages/MapPage/MapPage.js

```diff
@@ -60,7 +60,7 @@ const blurb = [
 
 const MapSubpage = ({ ...rest }) => (
   <DocsPageBuilder
-    title="Blockquote"
+    title="Map"
```

Comment:
😱

Reply:
Yeah my bad

Reply:
![png](https://user-images.githubusercontent.com/73652/40543309-b7ce03c0-601b-11e8-87ca-a077253f5147.png)

It me